common/views/components/FacilityPromo/FacilityPromo.js

```diff
@@ -44,47 +45,45 @@ const FacilityPromo = ({
       }}
       id={id}
       href={url}
-      className="plain-link promo-link"
     >
       <div>
         <div className="rounded-corners overflow-hidden">
           <UiImage {...uiImageProps} />
         </div>
 
-        <Space
-          v={{
-            size: 's',
-            properties: ['margin-top'],
-          }}
-          as="h2"
-          className={classNames({
-            'promo-link__title': true,
-            [font('wb', 4)]: true,
-          })}
-        >
-          {title}
-        </Space>
-        <p className={`${font('hnl', 5)} no-margin no-padding`}>
-          {description}
-        </p>
+        <CardBody>
```

Comment:
Why are we using `CardBody` and not just Card? Seems like a leaky abstraction somewhere?